modules/appeals_api/spec/services/appeals_api/pdf_construction/generator_spec.rb

```diff
@@ -116,123 +116,6 @@
       end
     end
 
```

Comment:
My goal is to eventually remove this file in favor of three API-specific files - I've worked on this file in the past and found it hard to navigate because it's so long and the appeal types are intermixed